git_externals/cli.py

```diff
@@ -226,6 +234,43 @@ def gitext_add(external, src, dst, branch, tag, ref):
         if dst not in git_externals[external]['targets'].setdefault(src, []):
             git_externals[external]['targets'][src].append(dst)
 
+    print_gitext_info(external, git_externals[external], root_dir='.')
+    dump_gitexts(git_externals)
+
+
+@cli.command('freeze')
+@click.argument('externals', nargs=-1, metavar='NAME')
+def gitext_freeze(externals):
+    """Freeze the externals revision"""
+    from git_externals import load_gitexts, dump_gitexts, foreach_externals_dir, root_path
+    git_externals = load_gitexts()
+    re_from_git_svn_id = re.compile("git-svn-id:.*@(\d+)")
+
+    def get_version(rel_url, ext_path, refs):
+        if 'tag' in refs:
```

Comment:
Here we are assuming the external repo is *svn*, then *git-svn* in case of failure, then *git*. Wouldn't it be more straightforward to use the new `vcs` field and directly goes for the correct way to retrieve the rev number. It may probably saves the user the time and/or the vision of error message (if any), or worse in case `svn` is not installed